docs/pages/versions/unversioned/sdk/audio.md

```diff
@@ -18,6 +18,24 @@ For [managed](../../introduction/managed-vs-bare/#managed-workflow) apps, you'll
 import { Audio } from 'expo-av';
 ```
 
+## Request recording permissions
+
+### `Audio.requestPermissionsAsync()`
+
+Asks the user to grant permissions for audio recording. Alias for `Permissions.askAsync(Permissions.AUDIO_RECORDING)`.
+
+#### Returns
+
+A promise that resolves to an object of type [PermissionsResponse](#permissionsresponse).
```

Comment:
I'm wondering if it isn't better to document `PermissionsResponse` type in Permissions docs and just link to that page here 🤔